### PR TITLE
feat: add translations support for plugin

### PR DESCRIPTION
**Pull request by mariajgrimaldi**
### Description
This PR adds translation support for es_ES and es-419.

### CODE REVIEW

File: openedx_unidigital/conf/locale/es_419/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+"to each team by selecting a team from the dropdown below and entering the "
+"instructor's username. Each instructor can be associated with multiple "
+"teams. When assigned to a team, the instructor will be able to manage the "
+"team's members."
+msgstr
+"Estos son los equipos asociados al curso. Puede asignar instructores "
+"a cada equipo seleccionando un equipo en el menú desplegable de abajo e introduciendo el "
+"nombre de usuario del instructor. Cada instructor puede estar asociado a varios "
+"equipos. Una vez asignado a un equipo, el instructor podrá gestionar los "
+"los miembros del equipo."
```
Comment (BryanttV) [inline openedx_unidigital/conf/locale/es_419/LC_MESSAGES/django.po]:
```suggestion
"miembros del equipo."
```

File: openedx_unidigital/conf/locale/es_ES/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+"to each team by selecting a team from the dropdown below and entering the "
+"instructor's username. Each instructor can be associated with multiple "
+"teams. When assigned to a team, the instructor will be able to manage the "
+"team's members."
+msgstr
+"Estos son los equipos asociados al curso. Puede asignar instructores "
+"a cada equipo seleccionando un equipo en el menú desplegable de abajo e introduciendo el "
+"nombre de usuario del instructor. Cada instructor puede estar asociado a varios "
+"equipos. Una vez asignado a un equipo, el instructor podrá gestionar los "
+"los miembros del equipo."
```
Comment (BryanttV) [inline openedx_unidigital/conf/locale/es_ES/LC_MESSAGES/django.po]:
```suggestion
"miembros del equipo."
```

File: openedx_unidigital/static/html/course_team_management.html
Comment (BryanttV) [inline openedx_unidigital/static/html/course_team_management.html:1]: Missing input placeholder to be marked for translation

**Review by BryanttV — CHANGES_REQUESTED** [review]

File: openedx_unidigital/conf/locale/es_419/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+msgid "Revoke Access"
+msgstr "Revocar acceso"
+
+#: static/html/course_team_management.html:45
+msgid ""
+"An error occurred. Please try again. If the problem persists, contact "
+"support."
+msgstr
+"Se ha producido un error. Por favor, inténtelo de nuevo. Si el problema persiste, póngase en contacto con "
+"con el servicio de soporte."
```
Comment (BryanttV) [inline openedx_unidigital/conf/locale/es_419/LC_MESSAGES/django.po]:
```suggestion
"el servicio de soporte."
```

File: openedx_unidigital/conf/locale/es_ES/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+msgid "Revoke Access"
+msgstr "Revocar acceso"
+
+#: static/html/course_team_management.html:45
+msgid ""
+"An error occurred. Please try again. If the problem persists, contact "
+"support."
+msgstr
+"Se ha producido un error. Por favor, inténtelo de nuevo. Si el problema persiste, póngase en contacto con "
+"con el servicio de soporte."
```
Comment (BryanttV) [inline openedx_unidigital/conf/locale/es_ES/LC_MESSAGES/django.po]:
```suggestion
"el servicio de soporte."
```

**Comment by mariajgrimaldi** [comment]
@BryanttV: I've addressed all your comments so I'm merging this now. Thank you!

**Pull request merged**